ui.content/src/main/content/jcr_root/conf/hclecomm/settings/wcm/policies/.content.xml

```diff
@@ -352,6 +352,19 @@
                         <jcr:content jcr:primaryType="nt:unstructured"/>
                     </policy_1587530274257>
                 </header>
+                <search jcr:primaryType="nt:unstructured">
+                    <policy_1589453975454
```

Comment:
@rakeshkumar-verma : where is this policy being used ?

Reply:
@rishabh-behl  updated template specific policies.